tests/test_mesh.py

```diff
@@ -2,12 +2,149 @@
 import trimesh
 import numpy as np
 import os
+from huggingface_hub import snapshot_download
 
 import genesis as gs
 import genesis.utils.mesh as mu
 import genesis.utils.gltf as gltf_utils
+import genesis.utils.usda as usda_utils
+from .utils import assert_allclose, assert_array_equal
 
+VERTICES_TOL = 1e-05  # Transformation loses a little precision in vertices
+NORMALS_TOL = 5e-02  # Conversion from .usd to .glb loses a little precision in normals
+UV_TOL = 1e-07  # Default tolerance for np.testing.assert_allclose
```

Comment:
This is not a good excuse. If you should match at machine precision, use machine precision: `gs.EPS`

Reply:
Changed it to gs.EPS